community/kernel/src/main/java/org/neo4j/kernel/recovery/PositionToRecoverFrom.java

```diff
@@ -47,18 +49,20 @@ public PositionToRecoverFrom( LatestCheckPointFinder checkPointFinder )
      * start recovery from
      * @throws IOException if log files cannot be read
      */
-    @Override
-    public LogPosition apply( long currentLogVersion ) throws IOException
+    public LogRecoveryInfo apply( long currentLogVersion ) throws IOException
```

Comment:
We are also returning the log recovery position which is not related to the checkpoint, so LogRecoveryInfo seems to make more sense at the moment.